Makefile

```diff
@@ -26,10 +26,6 @@ format: $(wildcard decide/*.?pp)
 
 
 ### Testing
-# Special case for utils file as it is only a header
-decide/utils.test: decide/utils.test.cpp
-	$(CXX) -o $@ $^ $(CXXFLAGS)
-
 %.test: %.test.cpp %.cpp
```

Comment:
`decide/utils.cpp` needs to be appended to this line as the test functionality depends on those functions being defined.

Reply:
Should be fixed now